exdb/templates/exdb/conclusion.html

```diff
@@ -6,7 +6,7 @@
         {% csrf_token %}
         <h1>Evaluation</h1>
         <h2>Experience: {{ experience.name }}</h2>
-        <p><a href="{% url 'ra_home' %}">Back to Home</a></p>
+        <p><a href="{% url 'home' %}">Return Home</a></p>
```

Comment:
I realize you probably weren't the one that did this, but we should probably put trans tags around this.